### PR TITLE
docs: OME-TIFF writing is now available

### PR DESCRIPTION
**Pull request by rleigh-dundee**
This should have been updated some time back, but was forgotten.

### CODE REVIEW

File: docs/sphinx/developers/cpp/overview.txt [hunk 1]
```diff
@@ -3,11 +3,10 @@ C++ overview
 
 A completely native Bio-Formats C++ interface is now available.
 Unlike the JACE bindings, this does not wrap the Java implementation.
-With this release, TIFF reading and writing, and OME-TIFF reading are
-available.  OME-TIFF writing will be available soon.  All other
-readers and writers from the Java implementation are currently
-unavailable; the intention is that support for these will be added
-over time.
+Currently readers and writers are provided for TIFF and OME-TIFF.
```
Comment (mtbc) [inline docs/sphinx/developers/cpp/overview.txt]: I wonder if we need this "Currently" especially as another one follows quickly.

Comment (rleigh-dundee) [inline docs/sphinx/developers/cpp/overview.txt]: It's documenting the current release, and we don't have immediate plans to add more readers so will likely not change for several releases.

Comment (mtbc) [inline docs/sphinx/developers/cpp/overview.txt]: Yes, I think that the following sentence covers that fact just fine though with its "currently". Happy for you to leave this sentence as it is if you prefer though.

**Comment by mtbc** [comment]
Good to merge as-is.

File: docs/sphinx/developers/cpp/overview.txt (hunk 1 above)
Comment (rleigh-dundee) [inline docs/sphinx/developers/cpp/overview.txt]: Pushed the change.

**Comment by mtbc** [comment]
:+1: Good to merge.

**Pull request merged**